imports/blocks/give/fieldsets/payment/index.js

```diff
@@ -0,0 +1,152 @@
+import { Component, PropTypes } from "react";
+
+import Validate from "../../../../util/validate";
+
+import Layout from "./Layout";
+
+export default class Payment extends Component {
+
+  static propTypes = {
+    save: PropTypes.func,
+    data: PropTypes.object,
+    savedAccount: PropTypes.object,
+    header: PropTypes.string,
+    children: PropTypes.object,
+    toggles: PropTypes.func,
+    transactionType: PropTypes.string,
+    schedules: PropTypes.object,
+    back: PropTypes.func,
+    next: PropTypes.func,
+  }
+
+  state = {
+    save: true,
+  }
+
+  savePayment = () => {
+    this.setState({ save: !this.state.save });
```

Comment:
@johnthepink should we adjust this to be a function that takes the previous state?

```js
this.setState(({ save }) => ({ save: !save }));
```